content/docs/command-reference/params/index.md

```diff
@@ -18,44 +18,44 @@ positional arguments:
 
 In order to track parameters and hyperparameters associated to machine learning
 experiments in <abbr>DVC projects</abbr>, DVC provides a different type of
-dependencies: _parameters_. Parameters are defined using the the `-p`
-(`--params`) option of `dvc run`, using simple names like `epochs`,
-`learning-rate`, `batch_size`, etc.
+dependencies: _parameters_. Params are defined in the `params` field of
```

Comment:
`or using ...` should be `corresponds to`

Reply:
Agree. This has already been updated: the `run` part is a parenthesis now.